utils/frame/try-runtime/cli/src/commands/follow_chain.rs

```diff
@@ -150,6 +150,10 @@ where
 			"TryRuntime_execute_block",
 			(block, command.state_root_check, command.try_state.clone()).encode().as_ref(),
 			full_extensions(),
+			shared
+				.export_proof
+				.as_ref()
+				.map(|path| path.as_path().join(&format!("{}-{}", number, hash))),
```

Comment:
```suggestion
				.map(|path| path.as_path().join(&format!("{}-{}.json", number, hash))),
```
or?

Reply:
Something with the file names is seriously wrong here :rofl:  
![image](https://user-images.githubusercontent.com/10380170/207970715-6cc9756c-993f-4fac-ad1e-bbdcac00e444.png)
Looks like it uses non-ASCII character '…' to produce `13371805-0x2dcb…bdc7.

Reply:
👀

Reply:
I think we don't really need the hash in the file name here, so I deleted it, that solves the problem.